src/Helpers/TermImportHelper.php

```diff
@@ -94,9 +105,10 @@ public function deleteTermAndTranslations($externalTerm)
     {
         collect($externalTerm->content_hub_ids)->each(function ($contentHubId) {
             if ($wpTermId = WpTerm::id_from_contenthub_id($contentHubId) ?? null) {
-                $this->preparePostRedirects($wpTermId);
-                wp_delete_term($wpTermId, $this->taxonomy);
-                $this->createPostRedirects();
+                $wpTerm = get_term($wpTermId);
+                if ($wpTerm instanceof \WP_Term) {
+                    $this->deleteTerm($wpTerm);
```

Comment:
@alfhen don't want to use the return value you just added, here?

Reply:
Nah its not needed in this case, and the return value has been there all along, but it is used another place in the script, so i just made sure to also return it after the refactor